test/test_regression_cli.py

```diff
@@ -54,17 +48,17 @@ def get_csv_md5(dataset, engines, tmpdir):
 
 def setup_module():
     """Update retriever scripts and cd to test directory to find data"""
-    os.chdir(os.path.dirname(os.path.abspath(__file__)))
+    os.chdir(rood_dir)
+    os.system('cp -r {0} {1}'.format(os.path.join(rood_dir, "test/raw_data"), rood_dir))
```

Comment:
we are keeping the raw_data in the test directory and we only copy it the retriever main directory.